packages/configs/ts/tsconfig.base.json

```diff
@@ -2,28 +2,28 @@
   "compilerOptions": {
     // https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports
```

Comment:
Why are we removing these?

Reply:
We are not, not sure why they're highlighted with dark red base but they are still there.

Reply:
I think github highlights them since comments aren't allowed in json?